### PR TITLE
Rename active_power_consumption to active_power in context of EVCharger

### LINKED ISSUE #15: Rename EvCharger ComponentData field `active_power_consumption` to `active_power`
To be consistent with other parts of the API,  but also because we'll soon support discharging from ev chargers,  so this field will not have just the power consumption.  https://github.com/frequenz-floss/frequenz-sdk-python/blob/v0.x.x/src/frequenz/sdk/microgrid/component_data.py#L206

### PR DESCRIPTION
**Pull request by jkozlowicz**

### CODE REVIEW

**Review by leandro-lucarella-frequenz — COMMENTED** [review]
Leaving this to @sahas-subramanian-frequenz , but this needs an entry in the `RELEASE_NOTES.md` if it goes in.

**Comment by jakub-toptal** [comment]
> Leaving this to @sahas-subramanian-frequenz , but this needs an entry in the `RELEASE_NOTES.md` if it goes in.

Done

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -6,7 +6,10 @@
 
 ## Upgrading
 
-<!-- Here goes notes on how to upgrade from previous versions, including deprecations and what they should be replaced with --> 
+<!-- Here goes notes on how to upgrade from previous versions, including deprecations and what they should be replaced with -->
+
+*  EVChargerData's `active_power_consumption` has been renamed to `active_power`
```
Comment (leandro-lucarella-frequenz) [inline RELEASE_NOTES.md]:
```suggestion
*  `EVChargerData`'s `active_power_consumption` has been renamed to `active_power`
```

**Review by leandro-lucarella-frequenz — COMMENTED** [review]
Thanks! Super tiny suggestion :D

File: RELEASE_NOTES.md (hunk 1 above)
Comment (jakub-toptal) [reply RELEASE_NOTES.md]: done

**Review by leandro-lucarella-frequenz — COMMENTED** [review]
Again, LGTM, so all yours @sahas-subramanian-frequenz for the final approve.

File: flaky-testing.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/zsh
+
+for i in `seq 1 1000`; do
+  pytest -v -s tests/power_distribution/test_power_distributor.py >/dev/null 2>&1
+  echo "$?"
+done
```
Comment (sahas-subramanian-frequenz) [inline flaky-testing.sh]: I guess this is a temporary script that you want to keep out of git?

Comment (jakub-toptal) [reply flaky-testing.sh]: Yep. Removed.

**Review by sahas-subramanian-frequenz — APPROVED** [review]

**Pull request merged**